tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -38,6 +38,34 @@ def test_format_number(number, expected, transformer):
     assert transformer().format_number(number) == expected
 
 
+@pytest.mark.parametrize(
+    "value, expected",
+    (
+        (123, "123%"),
```

Comment:
A decimal object should be tested.

Reply:
We already are testing decimal here, I added `("123.4", "123.4%"),("123.40", "123.40%")`, I added `(0.00, "0%")` as well.

Reply:
I mean, a `Decimal` object, `"123.4"` is just a string. We shouldn't get floats like `0.00` we should get `Decimal` object representing those.

Reply:
Done thanks!

Reply:
Ok, we have a Decimal test in test format unit now and here.